e2e/cypress/e2e/runner/MiniSummaryPageController.feature

```diff
@@ -0,0 +1,8 @@
+# Created by calum-ukhsa at 29/07/2025
+Feature: MiniSummaryPageController
+
+  @wip
```

Comment:
We can't accept work in progress to main - with no control over if the work ever gets done this is just accepting debt that could never be paid back. Tests must be implemented prior to merging.